includes/Integrations/Site_Kit.php

```diff
@@ -76,6 +77,23 @@ protected function is_plugin_active() {
 		return defined( 'GOOGLESITEKIT_VERSION' );
 	}
 
+
+	/**
+	 * Determines whether the built-in adsense module in Site Kit is active.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return bool Whether Site Kit's analytics module is active.
+	 */
+	protected function is_adsense_module_active() {
+		$adsense_module_active       = in_array( 'adsense', $this->get_site_kit_active_modules_option(), true );
+		$adsense_options             = get_option( 'googlesitekit_adsense_settings' );
+		$adsense_options_use_snippet = ! empty( $adsense_options['useSnippet'] );
+		$adsense_web_stories_ad_unit = ! empty( $adsense_options['webStoriesAdUnit'] );
+
+		return $adsense_module_active && $adsense_options_use_snippet && $adsense_web_stories_ad_unit;
+	}
```

Comment:
Neat!

Reply:
I would love to get someone from Sitekit team to validate that this check is good. I based it on this [file](https://github.com/google/site-kit-wp/blob/d6ea05554e20353df9e770abd193c98b006cdf79/includes/Modules/AdSense/Tag_Guard.php). @felixarntz does this check seem right to you?

Reply:
I wonder whether we'd need to check for `clientID` existence as well 🤔  After all, story ads need both a client ID and an ad unit ID.

Reply:
I added a check for `clientID` in https://github.com/google/web-stories-wp/pull/7622/commits/f71751edc64a2158be5d17e7f29f0a0ca435d54a. I can't see what is hurts to check for this..